img/e/epo.org.svg

```diff
@@ -0,0 +1 @@
a5.8 5.8 0 0 0 1 .1c1.28 0 2.14-.65 2.14-1.61s-.71-1.19-1.28-1.4zm37.49 0c-.46-.16-.85-.29-.85-.69s.3-.58.84-.58a3.36 3.36 0 0 1 1 .14v-.91a6 6 0 0 0-.92-.09 2.31 2.31 0 0 0-1.64.55 1.4 1.4 0 0 0-.4 1c0 1 .69 1.2 1.3 1.4.45.15.84.27.84.68a.5.5 0 0 1-.15.37 1.11 1.11 0 0 1-.78.25 4 4 0 0 1-1.06-.15v.94a6 6 0 0 0 1 .1c1.28 0 2.15-.65 2.15-1.61s-.71-1.19-1.33-1.4zm-28.12-1.67a1.7 1.7 0 0 0-1.23-.46 1.63 1.63 0 0 0-1.51.89 7.55 7.55 0 0 0 .06-.8V74h-1.1v7.08a6.46 6.46 0 0 0 1.75.21A2.5 2.5 0 0 0 121 78.5a2.55 2.55 0 0 0-.63-1.86zm-1 3.43a1.3 1.3 0 0 1-.94.34 2.22 2.22 0 0 1-.73-.1v-1.49a2.37 2.37 0 0 1 .3-1.17 1 1 0 0 1 .88-.54.78.78 0 0 1 .58.22 1.78 1.78 0 0 1 .35 1.28 2 2 0 0 1-.45 1.46zm5.79-3.89a1.48 1.48 0 0 0-1.58 1 6.77 6.77 0 0 0 .07-.91h-1.06v4.93h1.11v-2a2.29 2.29 0 0 1 .51-1.76 1.1 1.1 0 0 1 .9-.18v-1.08zm9.67.1l-.94 2.64c-.11.32-.2.65-.26.89a9.1 9.1 0 0 0-.3-.93l-.91-2.6h-1.22l1.85 4.93h1.1l1.85-4.93zm9.17 0v-1.84l-1.09.3v1.54h-.94v.87h.94v2.63a1.64 1.64 0 0 0 .33 1.21 1.4 1.4 0 0 0 1.05.31 4.89 4.89 0 0 0 1-.11v-.92a2.53 2.53 0 0 1-.63.08.58.58 0 0 1-.44-.13c-.13-.12-.17-.35-.17-.77v-2.3h1.27v-.87zM41.07 49.75c1.42-3.8 1.72-7.24.77-9a2.11 2.11 0 0 0-1.41-1.14c-2.31-.53-5.55 2.83-7.38 7.66-1.39 3.66-1.71 6.89-.84 8.65a2.32 2.32 0 0 0 1.43 1.27c2.3.68 5.63-2.66 7.43-7.44zm5 1.17A20.84 20.84 0 0 1 38 61.25a8.73 8.73 0 0 1-6.88 1.31 6.36 6.36 0 0 1-4.1-3.46c-1.54-3.13-1.18-8 .9-13.25C31 38 37.37 32.88 42.42 34.12a6.2 6.2 0 0 1 4.09 3.29c1.73 3.26 1.58 8.18-.41 13.51z"/><path d="M42.56 24.24A2.63 2.63 0 0 1 45.19 23 9.29 9.29 0 0 1 49 24.73a17.57 17.57 0 0 1 5.76 7.63c2.59 6.07 3.93 15.18-2 25.89-4 7.31-9.56 11.65-14.64 13.37a23.32 23.32 0 0 1-10.6.95 16.94 16.94 0 0 1-4.67-1.31 11.35 11.35 0 0 1-2.54-1.52c-1.87-1.57-1.9-3-1.3-4.18a2.64 2.64 0 0 1 2.79-1.46 4.08 4.08 0 0 1 1.75.7 12.06 12.06 0 0 0 2.34 1.29 15 15 0 0 0 10 .12c4.23-1.31 8.81-5.27 11.91-11A27.89 27.89 0 0 0 51 44.93a18.47 18.47 0 0 0-3.21-13.27A15.39 15.39 0 0 0 43.34 28a2.82 2.82 0 0 1-.78-3.76z"/><path d="M19.68 75.28A15.36 15.36 0 0 0 23 76.63a27.2 27.2 0 0 0 8.86.68c7.62-.54 17.31-4.9 23.76-16.79 6.55-12.07 5.61-23.8 1.12-32.84a23.18 23.18 0 0 0-9.11-9.53 20.55 20.55 0 0 0-4.62-2 2.69 2.69 0 0 1-1.89-3.66 2.82 2.82 0 0 1 3-1.9 9.87 9.87 0 0 1 2.89.66 25.24 25.24 0 0 1 4.26 2.14A30.24 30.24 0 0 1 63 27.06C67.45 37 68 50.19 61 63 48.29 86.1 26.48 85.28 18.29 81.28A11.31 11.31 0 0 1 14.92 79a2.71 2.71 0 0 1-.38-3.49 2.55 2.55 0 0 1 2-1.28 3.39 3.39 0 0 1 1.21.12 4.58 4.58 0 0 1 1 .46c.17.05.48.19.93.47z"/><path d="M62.87 66.37a4.9 4.9 0 0 1 .3-.85c.11-.24.23-.49.4-.79.38-.64.75-1.35 1-1.85 2.9-5.74 6.23-15.52 5-25.5-1.25-10.52-5-18.71-11.57-24.23a29.38 29.38 0 0 0-2.49-1.87 5.76 5.76 0 0 1-1.16-.94A2.71 2.71 0 0 1 54.12 7c1.11-1.87 3-1.7 5.56.08a33.85 33.85 0 0 1 6.67 6.3c3.74 4.56 7.79 11.86 9.05 23.11a48.44 48.44 0 0 1-1.31 17.21 57.44 57.44 0 0 1-4.15 11.56c-.42.83-.86 1.57-1.24 2.19a5.32 5.32 0 0 1-1.53 1.71 3.12 3.12 0 0 1-3.27-.06 2.38 2.38 0 0 1-1.03-2.73z"/></g></svg>
```

Comment:
As the image you provided seems more of a logo than an icon, I think this version works better:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 90 90" style="background:#be0f05;fill:#fff"><path d="M62.87 66.37a4.9 4.9 0 0 1 .3-.85c.11-.24.23-.49.4-.79.38-.64.75-1.35 1-1.85 2.9-5.74 6.23-15.52 5-25.5-1.25-10.52-5-18.71-11.57-24.23a29.38 29.38 0 0 0-2.49-1.87 5.76 5.76 0 0 1-1.16-.94A2.71 2.71 0 0 1 54.12 7c1.11-1.87 3-1.7 5.56.08a33.85 33.85 0 0 1 6.67 6.3c3.74 4.56 7.79 11.86 9.05 23.11a48.44 48.44 0 0 1-1.31 17.21 57.44 57.44 0 0 1-4.15 11.56c-.42.83-.86 1.57-1.24 2.19a5.32 5.32 0 0 1-1.53 1.71 3.12 3.12 0 0 1-3.27-.06 2.38 2.38 0 0 1-1.03-2.73Zm-43.19 8.91A15.36 15.36 0 0 0 23 76.63a27.2 27.2 0 0 0 8.86.68c7.62-.54 17.31-4.9 23.76-16.79 6.55-12.07 5.61-23.8 1.12-32.84a23.18 23.18 0 0 0-9.11-9.53 20.55 20.55 0 0 0-4.62-2 2.69 2.69 0 0 1-1.89-3.66 2.82 2.82 0 0 1 3-1.9 9.87 9.87 0 0 1 2.89.66 25.24 25.24 0 0 1 4.26 2.14A30.24 30.24 0 0 1 63 27.06C67.45 37 68 50.19 61 63 48.29 86.1 26.48 85.28 18.29 81.28A11.31 11.31 0 0 1 14.92 79a2.71 2.71 0 0 1-.38-3.49 2.55 2.55 0 0 1 2-1.28 3.39 3.39 0 0 1 1.21.12 4.58 4.58 0 0 1 1 .46c.17.05.48.19.93.47zm22.88-51.04A2.63 2.63 0 0 1 45.19 23 9.29 9.29 0 0 1 49 24.73a17.57 17.57 0 0 1 5.76 7.63c2.59 6.07 3.93 15.18-2 25.89-4 7.31-9.56 11.65-14.64 13.37a23.32 23.32 0 0 1-10.6.95 16.94 16.94 0 0 1-4.67-1.31 11.35 11.35 0 0 1-2.54-1.52c-1.87-1.57-1.9-3-1.3-4.18a2.64 2.64 0 0 1 2.79-1.46 4.08 4.08 0 0 1 1.75.7 12.06 12.06 0 0 0 2.34 1.29 15 15 0 0 0 10 .12c4.23-1.31 8.81-5.27 11.91-11A27.89 27.89 0 0 0 51 44.93a18.47 18.47 0 0 0-3.21-13.27A15.39 15.39 0 0 0 43.34 28a2.82 2.82 0 0 1-.78-3.76Zm-1.49 25.51c1.42-3.8 1.72-7.24.77-9-.27-.57-.8-1-1.41-1.14-2.31-.53-5.55 2.83-7.38 7.66-1.39 3.66-1.71 6.89-.84 8.65.27.6.8 1.07 1.43 1.27 2.3.68 5.63-2.66 7.43-7.44zm5 1.17A20.84 20.84 0 0 1 38 61.25a8.73 8.73 0 0 1-6.88 1.31 6.36 6.36 0 0 1-4.1-3.46c-1.54-3.13-1.18-8 .9-13.25C31 38 37.37 32.88 42.42 34.12a6.2 6.2 0 0 1 4.09 3.29c1.73 3.26 1.58 8.18-.41 13.51z"/></svg>
```

Reply:
Thanks looks very good.